browser/src/react-redux/App/LoginPage.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { loginSuccess, loginFailure } from './User/updateUser.action'
+import axios from 'axios'
+
+const initialState = {
+  email: '',
+  password: '',
+  generalError: ''
+}
+
+class LoginPage extends React.Component {
+  state = initialState
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  onSubmit = e => {
+    e.preventDefault()
+    this.setState(initialState)
```

Comment:
Why are we resetting here? Keep in mind that this would also reset when the login fails. We want to give the users a chance to edit their input first. It can be frustrating to just have all of your input cleared out on each error.

Reply:
Do we need to reset at all? singupPage and loginPage are setup to have a empty initial state when they are mounted. Both components redirect after successful post request, which unmounts them. If you travel back to signup or login, both states will be empty. Am I thinking about this the right way?

Reply:
Sorry, I missed these comments last night. We definitely do not need to reset here. You did the right thing by removing it.